s4e-web/src/app/errors/errors.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ErrorsComponent } from './errors.component';
+
+describe('ErrorComponent', () => {
+  let component: ErrorsComponent;
+  let fixture: ComponentFixture<ErrorsComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ ErrorsComponent ]
+    })
```

Comment:
It's better to import whole module which contains component. This way you make sure, that the module itself has all the dependencies for the component. In this case this is rather trivial, but that's the general rule. And it makes sure that as you add dependencies to the module your tests won't *desynchronize*

Reply:
It has been generated automatically...